ansible_base/authentication/serializers/authenticator.py

```diff
@@ -78,6 +78,14 @@ def to_internal_value(self, data):
         return parsed_data
 
     def validate(self, data) -> dict:
+        # prevent disabling the last enabled authenticator
+        if data.get('enabled') is False:
+            if self.instance and self.instance.enabled:
+                # count how many other authenticators are enabled ...
+                qs = Authenticator.objects.filter(enabled=True).exclude(id=self.instance.id)
```

Comment:
It might be good to combine the two querysets into an instance method `Authenticator.is_the_last_airbender`